portfolio/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title> ksenia iorzh junior web develorer </title>
+        <link rel="stylesheet" href="styles.css" />
+    </head>
+    <body>
+        <header>
+            <nav>
+                <ul>
+                    <li><a href="#about" id="about-link">about</a></li>
+                    <li><a href="#skills" id="skills-link">skills</a></li>
```

Comment:
The color change on link hover is a good way to provide visual feedback to users. However, you might want to experiment with different types of hover effects (such as underline, background color change, etc.) to make the interaction more engaging